<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>YDPlayerMainWidget</class>
 <widget class="QWidget" name="YDPlayerMainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1084</width>
    <height>610</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1084</width>
    <height>610</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>C++Player</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#YDPlayerMainWidget
{
background:#45484E;
}</string>
  </property>
  <widget class="QFrame" name="frameBottom">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>520</y>
     <width>477</width>
     <height>48</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>48</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>48</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>30</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSlider">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>8</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>8</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QSlider::groove:horizontal {
border:none;
background:none;
height: 4px;
border-radius:2px;
padding-left:-1px;
padding-right:-1px;
}

QSlider::sub-page:horizontal {
background:rgba(246, 76, 40, 1);
border:none;
height: 4px;
border-radius: 2px;
}

QSlider::add-page:horizontal {
background:rgba(153, 153, 153, 0.6);
border:none;
height:4px;
border-radius: 2px;
}

QSlider::handle:horizontal 
{
    background:rgba(255, 255, 255, 1);
    width:6px;
	height:6px;
	margin-top:-1px;
	margin-bottom:-1px;
    border-radius:3px;
}

QSlider::handle:horizontal:hover 
{
    background:rgba(255, 255, 255, 1);
    width:8px;
	height:8px;
    border-radius:4px;
	margin-top:-1px;
	margin-bottom:-1px;
}



QSlider::groove:horizontal:hover {
border:none;
background:none;
height: 6px;
border-radius:3px;
padding-left:-1px;
padding-right:-1px;
}

QSlider::sub-page:horizontal:hover {
background:rgba(246, 76, 40, 1);
border:none;
height: 6px;
border-radius: 3px;
}

QSlider::add-page:horizontal:hover {
background:rgba(153, 153, 153, 0.6);
border:none;
height:6px;
border-radius: 3px;
}</string>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>30</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>30</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonPlay">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>28</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>28</width>
          <height>32</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
	background:none;
	border:none;
	image: url(:/images/icon_play.png);
}

QPushButton:hover
{
	background:none;
	border:none;
	image: url(:/images/icon_play_hover.png);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>70</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="labelTime">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>88</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel
{
color:rgba(255, 255, 255, 1);
font-family:&quot;Microsoft YaHei&quot;;
font-size:14px;
border:none;
background:none;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonSpeaker">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>100</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
	background:none;
	border:none;
	image: url(:/images/icon_speaker.png);
}

QPushButton:hover
{
	background:none;
	border:none;
	image: url(:/images/icon_speaker_hover.png);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>175</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frameVolume">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>190</y>
     <width>36</width>
     <height>140</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>36</width>
     <height>140</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>36</width>
     <height>140</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#frameVolume
{
	background:rgba(51, 51, 51, 0.9);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>4</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>15</number>
    </property>
    <item>
     <widget class="QLabel" name="labelVolume">
      <property name="styleSheet">
       <string notr="true">QLabel
{
color:rgba(255, 255, 255, 1);
font-family:&quot;Microsoft YaHei&quot;;
font-size:12px;
border:none;
background:none;
}</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="verticalSlider">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QSlider::groove:vertical {
border:none;
background:none;
width: 4px;
border-radius: 2px;
padding-left:-1px;
padding-right:-1px;
padding-top:0px;
padding-bottom:0px;
}

QSlider::sub-page:vertical {
background:rgba(34, 34, 34, 1);
border:none;
border-radius: 2px;
width:4px;
}

QSlider::add-page:vertical {
background:rgba(15, 134, 255, 1);
border:none;
border-radius: 2px;
width:4px;
}

QSlider::handle:vertical 
{
	background:white;
	width:16px;
    height: 16px;
    margin-left: -6px;
    margin-right: -6px;
    border-radius: 8px;
}</string>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButtonReplay">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>270</y>
     <width>51</width>
     <height>52</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>51</width>
     <height>52</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>51</width>
     <height>52</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background:none;
	border:none;
	image: url(:/images/icon_replay.png);
}

QPushButton:hover
{
	background:none;
	border:none;
	image: url(:/images/icon_replay_hover.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOpenFile">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
color:#666666;
background:#FFFFFF;
font-family:&quot;Microsoft YaHei&quot;;
font-size: 14px;
border:1px solid #AAAAAA;
border-radius: 14px;
}

QPushButton:hover
{
color:#FFFFFF;
background:#0F88FF;
font-family:&quot;Microsoft YaHei&quot;;
font-size: 14px;
border-radius: 14px;
}</string>
   </property>
   <property name="text">
    <string>打开文件</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
